### DISCUSSION TITLE
Requisite Libraries for CDP

### DISCUSSION THREAD

**Discussion by tqrahman** [Code-and-Response/ClusterDuck-Protocol#336] [QuackOverflow] [answered]
When I clone the ClusterDuck Protocol repo, the libraries are not cloned. Is there an easy way to get the libraries?

**Comment by ShivankAg** [comment, marked as answer]
You can pull the libraries down by running the python script (install-dependencies.py) here: https://github.com/Call-for-Code/ClusterDuck-Protocol/tree/master/cdp-dependency-automator-script

Make sure you install the requests library first (pip install requests) before running the script (python install-dependencies.py)
